autogen/variables.tf.tmpl

```diff
@@ -107,6 +107,11 @@ variable "update_policy" {
 # Healthcheck
 ##############
 
+variable "health_check_name" {
+  description = "Health check name. When set to `default`, name will be derived from var.hostname."
```

Comment:
Instead of using `default` as a special name, we should use `""` as the default (where it will be derived from var.hostname).

Reply:
@morgante shall I change `mig_name` to empty string instead "default" as well?

Reply:
Yes, please do. Thanks!